7.1/contrib/etc/conf.d/php-fpm/pool.conf

```diff
@@ -17,14 +17,15 @@ ping.path = /ping
 
 ping.response = pong
 
-access.log = /tmp/www.access.log
+access.log = /proc/self/fd/2
+access.format = %t \”%m %r%Q%q\” %s %{mili}d %{kilo}M %C%%”
 
+slowlog = /proc/self/fd/2
+request_slowlog_timeout = 10s
 
-slowlog = /tmp/www-slow.log
-
-php_admin_value[error_log] = /tmp/www-error.log
+php_admin_value[error_log] = /proc/self/fd/2
 php_admin_flag[log_errors] = on
-php_admin_value[memory_limit] = 256M
+php_admin_value[memory_limit] = 510M
```

Comment:
openshift online default quota is 512MiB. moved close to that and have seen java processes get killed when they hit quota that took a while to figure out why so kept it a bit lower as that was a real time waste to debug.